src/admin/seeds/dashboard/dashboard.ts

```diff
@@ -8,6 +8,7 @@ import { EventAggregator } from "aurelia-event-aggregator";
 import { DisposableCollection } from "services/DisposableCollection";
 import { EventConfigException } from "services/GeneralEvents";
 import { WhiteListService } from "services/WhiteListService";
+import { BigNumber } from "@ethersproject/providers/node_modules/@ethersproject/bignumber";
```

Comment:
Should be like in the rest of our project, and not dependent on package path:

`import { BigNumber } from "ethers";`